client/components/download-link/renderers/docx-renderer.js

```diff
@@ -875,16 +875,19 @@ export default (application, sections, values, updateImageDimensions) => {
         return Promise.resolve(value);
       }
 
+      let valueWasJson = false;
+
       if (typeof value === 'string') {
         try {
           value = JSON.parse(value);
+          valueWasJson = true;
         } catch (e) {
           return Promise.resolve(value);
         }
       }
 
       if (!value.document || !value.document.nodes) {
-        return Promise.resolve(value);
```

Comment:
If the value was a JSON string, and get's parsed to an object on line 880, then returning the object and not the original value here was causing the species not to render.

I don't know if line 901 is also causing similar issues, but I thought that it was best to always return the value in the original format.